### PR TITLE
[dynamo+aten] Enable embedding_bag_byte_rowwise_offsets + meta kernel impl

### PR DESCRIPTION
**Pull request by jfix71**
Differential Revision: D47007550

cc @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10

### CODE REVIEW

File: aten/src/ATen/native/quantized/cpu/qembeddingbag.cpp [hunk 1]
```diff
@@ -957,6 +957,29 @@ Tensor embedding_bag_2bit_rowwise_offsets(
   return output;
 }
 
+Tensor embedding_bag_byte_rowwise_offsets_meta(
+    const Tensor& weight,
+    const Tensor& /* indices */,
+    const c10::optional<Tensor>& offsets_in,
+    const bool /* scale_grad_by_freq */,
+    const int64_t /* mode */,
```
Comment (gmagogsfm) [inline aten/src/ATen/native/quantized/cpu/qembeddingbag.cpp:965]: Can you add asserts to make sure the assumptions to made on arguments like `mode` are explicitly enforced?

Comment (jfix71) [reply aten/src/ATen/native/quantized/cpu/qembeddingbag.cpp:965]: @gmagogsfm Updated:
* Validated that `mode` and `scale_grad_by_freq` [aren't used at all in the actual op](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/quantized/cpu/qembeddingbag.cpp#L889-L890)
* Brought over some checks for `indices` based on ndims/dtypes, and also offsets dtype
* Validated `per_sample_weights_`, `pruned_weights`, and `compressed_indices_mapping` make no difference on the output -- entirely internal to the kernel.

**Review by gmagogsfm — APPROVED** [review]
I don't know the op semantics well, so didn't check the correctness of shape fn.

File: aten/src/ATen/native/quantized/cpu/qembeddingbag.cpp [hunk 2]
```diff
@@ -957,6 +957,45 @@ Tensor embedding_bag_2bit_rowwise_offsets(
...
+    bool /* pruned_weights */,
+    const c10::optional<Tensor>& /* per_sample_weights_ */,
+    const c10::optional<Tensor>& /* compressed_indices_mapping */,
+    bool include_last_offset) {
+  TORCH_CHECK(
+      indices.dim() == 1 || indices.dim() == 2,
+      "qembedding/qembedding_bag operator supports 1 or 2d indices, got ",
```
[the remark this answers is not in the document]
Comment (jfix71) [reply aten/src/ATen/native/quantized/cpu/qembeddingbag.cpp:972]: I actually copied this over from elsewhere and thought it'd be good to match 😄 but sure I can change it in this new code.
https://github.com/pytorch/pytorch/blob/02abbb8109a98c2f0a6e01d0ea066731a30acefb/aten/src/ATen/native/quantized/cpu/qembeddingbag.cpp#L534

**Pull request closed without merging**